ARM/ARM-Template-Deployment.yml

```diff
@@ -0,0 +1,55 @@
+name: workflow to deploy an ARM Template using CLI commands
+on: push
+
+# CONFIGURATION
+# For help, go to https://github.com/Azure/Actions
+#
+# Set up the following secrets in your repository:
+#   AZURE_CREDENTIALS
+# Change these variables for your configuration:
```

Comment:
Lets specify that these are env variables. Add a link to GH Actions doc talking about env variables.